### ISSUE TITLE
Is this project offline

### ISSUE THREAD

**Issue by TomlDev** [FlorianFFF/RT3Multiplayer#2]
Can you bring it back to work? 👍🏽 
Or release the code?

**Comment by CJman1** [comment]
Bumping this. Seems to be a dead link `http://florianf.xyz/getip?software=Railty3MP` which prevents the launcher from loading.

**Comment by TomlDev** [comment]
Yes, sadly. I tried to contact the developer of the project via mail and he told me that he would bring back the server but now the version is incompatible it says...
